packages/flutter_tools/bin/fuchsia_tester.dart

```diff
@@ -70,16 +81,37 @@ Future<Null> run(List<String> args) async {
     if (!fs.isFileSync(shellPath)) {
       throwToolExit('Cannot find Flutter shell at $shellPath');
     }
-    loader.installHook(
-      shellPath: shellPath,
-    );
+    // Put the tester shell where runTests expects it.
+    final Link testerDestLink =
```

Comment:
Add a TODO to switch to a Fuchsia-specific `Artifacts` implementation.

Reply:
Done.